text/message_validation.go

```diff
@@ -164,6 +164,15 @@ func NewErrorValidationDuplicateCredentials() *Message {
 	}
 }
 
+func NewErrorValidationDuplicateCredentialsCustomMessage(reason string) *Message {
+	return &Message{
+		ID:      ErrorValidationDuplicateCredentials,
```

Comment:
Consider adding a new ID for this message.

Reply:
I agree, this needs to be marked as deprecated somehow and then we need to have this new error message have its own ID. 

we don't know if customers rely on the errors ID in their codebase to make the correct decision.